src/main/fc/fc_tasks.c

```diff
@@ -494,7 +494,7 @@ cfTask_t cfTasks[TASK_COUNT] = {
     [TASK_BARO] = {
         .taskName = "BARO",
         .taskFunc = taskUpdateBaro,
-        .desiredPeriod = TASK_PERIOD_HZ(20),
+        .desiredPeriod = TASK_PERIOD_MS(50),        // SPL06 uses oversampilng - result 16ms ?
```

Comment:
20Hz ends up being 50ms. Is this change needed?

Reply:
Not really - it can be dropped. I just did this modification because it was easier for me to understand difference between what baro wants and what it gets (it interfered with pitot measurements since the true period for SPL06 is 16ms).

Reply:
@jmajew please remove it if it's not required for those changes

Reply:
Done.